CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -4,21 +4,55 @@ Customization guide
 The Gini Capture SDK components can be customized either through the `GiniConfiguration`, the `Localizable.string` file or through the assets. Here you can find a complete guide with the reference to every customizable item.
 
 - [Generic components](#generic-components)
+- [Onboarding screens](#onboarding-screens)
+- [No Authorized screen](#no-authorized-screen)
 - [Camera screen](#camera-screen)
+- [Gallery album screen](#gallery-album-screen)
 - [Review screen](#review-screen)
-- [Multipage Review screen](#multipage-review-screen)
 - [Analysis screen](#analysis-screen)
-- [Supported formats screen](#supported-formats-screen)
-- [Open with tutorial screen](#open-with-tutorial-screen)
-- [Capturing tips screen](#capturing-tips-screen)
-- [Gallery album screen](#gallery-album-screen)
-- [Onboarding screens](#onboarding-screens)
-- [Help screen](#help-screen)
+- [Help screens](#help-screens)
+- [No result screen](#no-result-screen)
+- [Error screens](#error-screens)
+
+## Colors
+
+We are providing a global color palette `GiniColors.xcassets` which you are free to override. The custom colors will be then applied on all screens.
+You can find the names of the colors [https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets](https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets).
+
+ You can view our color palette here:
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FH4WFJ4xaw4YNU4VaJYWiQq%2FiOS-Gini-Capture-SDK-2.0.0-UI-Customisation%3Fnode-id%3D14%253A355%26t%3DwpenBBM8QsagJzOg-1" allowfullscreen></iframe>
+
+## Typography
+
+We provide a global typography based on text appearance styles from `UIFont.TextStyle`. 
 
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FH4WFJ4xaw4YNU4VaJYWiQq%2FiOS-Gini-Capture-SDK-2.0.0-UI-Customisation%3Fnode-id%3D123%253A2326%26t%3DwpenBBM8QsagJzOg-1" allowfullscreen></iframe>
```

Comment:
[Suggestion] Same

Reply:
I'll keep with the Figma preview. You can check the latest documentation [version](https://github.com/gini/gini-mobile-ios/actions/runs/3513599964).